docs/content/configuration/storage.md

```diff
@@ -114,110 +115,151 @@ After this you can pass you credentials inside `config.toml` file
-Also configuration for this storage looks almost the same in our proxy as for [Minio](#minio). 
+Also configuration for this storage looks almost the same in our proxy as for [Minio](#minio).
 
 ##### Configuration:
 
     # StorageType sets the type of storage backend the proxy will use.
     # Env override: ATHENS_STORAGE_TYPE
     StorageType = "minio"
-    
-    [Storage] 
+
+    [Storage]
         [Storage.Minio]
             # Address of DO Spaces storage
             # Env override: ATHENS_MINIO_ENDPOINT
             Endpoint = "YOUR_ADDRESS.digitaloceanspaces.com"
-    
+
             # Access Key for Minio storage
             # Env override: ATHENS_MINIO_ACCESS_KEY_ID
             Key = "YOUR_DO_SPACE_KEY_ID"
-    
+
             # Secret Key for DO Spaces storage
             # Env override: ATHENS_MINIO_SECRET_ACCESS_KEY
             Secret = "YOUR_DO_SPACE_SECRET_KEY"
-    
+
             # Timeout for networks calls made to DO Spaces storage in seconds
             # Defaults to Global Timeout
             Timeout = 300
-    
-            # Enable SSL 
+
+            # Enable SSL
             # Env override: ATHENS_MINIO_USE_SSL
             EnableSSL = true
-    
+
             # Space name in your DO Spaces storage
             # Env override: ATHENS_MINIO_BUCKET_NAME
             Bucket = "YOUR_DO_SPACE_NAME"
-            
+
             # Region for Minio storage
             # Env override: ATHENS_MINIO_REGION
             Region = "YOUR_DO_SPACE_REGION"
-            
\ No newline at end of file
+
+## Alibaba OSS
+
+For Athens to communicate with [Alibaba Cloud Object Storage Service](https://www.alibabacloud.com/product/oss), we are using Minio driver.
+Also configuration for this storage looks almost the same in our proxy as for [Minio](#minio).
+
+##### Configuration:
+
+    # StorageType sets the type of storage backend the proxy will use.
+    # Env override: ATHENS_STORAGE_TYPE
+    StorageType = "minio"
+
+    [Storage]
+        [Storage.Minio]
+            # Address of DO Spaces storage
+            # Env override: ATHENS_MINIO_ENDPOINT
+            Endpoint = "YOUR_ADDRESS.aliyuncs.com"
+
+            # Access Key for Minio storage
+            # Env override: ATHENS_MINIO_ACCESS_KEY_ID
+            Key = "YOUR_OSS_KEY_ID"
+
+            # Secret Key for DO Spaces storage
+            # Env override: ATHENS_MINIO_SECRET_ACCESS_KEY
+            Secret = "YOUR_OSS_SECRET_KEY"
+
+            # Timeout for networks calls made to OSS storage in seconds
+            # Defaults to Global Timeout
+            Timeout = 300
+
+            # Enable SSL
+            # Env override: ATHENS_MINIO_USE_SSL
+            EnableSSL = true
+
+            # Space name in your DO Spaces storage
+            # Env override: ATHENS_MINIO_BUCKET_NAME
+            Bucket = "YOUR_OSS_BUCKET_NAME"
+
+            # Region for Minio storage
+            # Env override: ATHENS_MINIO_REGION
+            Region = "YOUR_OSS_REGION"
```

Comment:
We probably could remove that from this section. Checked that and without passing region everything works. It's probably because user provide region in endpoint eg. `oss-eu-central-1.aliyuncs.com`

Reply:
Hmm, I'll look further into if it's actually used anywhere. I agree, if it's not used we should drop it. :+1:

Reply:
After reading more of the docs, I agree with you, we should drop the Location for Alibaba OSS.

From the docs, the bucket region is fixed to the API endpoint region, so you need to change the endpoint to change the region.

The [PUT docs](https://www.alibabacloud.com/help/doc-detail/31959.htm) say:
> The region of the created bucket is consistent with the region of the endpoint from which the request is sent. Once the data center of the bucket is determined, all objects in this bucket are stored in the corresponding region. For more information, see Regions and endpoints .